examples/kind/test/local-path-provisioner.yaml

```
@@ -60,7 +60,7 @@ spec:
       serviceAccountName: local-path-provisioner-service-account
       containers:
       - name: local-path-provisioner
-        image: rancher/local-path-provisioner:v0.0.5
+        image: rancher/local-path-provisioner:v0.0.6
```

Looks like there's another version out?

```suggestion
        image: rancher/local-path-provisioner:v0.0.7
```

---

Now https://github.com/rancher/local-path-provisioner/releases/tag/v0.0.8. Moving pretty fast

---

0.0.9 :) updated